<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="verticalGroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>600</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1100</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Invention App</family>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;APPRENEZ L'HISTOIRE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ink Free</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>FLASHCARD LEARNER</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>150</width>
        <height>200</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QFrame" name="horizontalFrame">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Invention App</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Filtres actuels :</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="currentAreaLabel">
         <property name="font">
          <font>
           <family>Invention App</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Invention App</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="currentCategoryLabel">
         <property name="font">
          <font>
           <family>Invention App</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="learnBtn">
        <property name="text">
         <string>Apprendre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="testBtn">
        <property name="text">
         <string>Se tester</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="filterBtn">
        <property name="text">
         <string>SÃ©lectionner des filtres</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
